qualtran/bloqs/arithmetic/comparison.py

```diff
@@ -1154,3 +1156,423 @@ def _clineardepthgreaterthan_example() -> CLinearDepthGreaterThan:
 _CLinearDepthGreaterThan_DOC = BloqDocSpec(
     bloq_cls=CLinearDepthGreaterThan, examples=[_clineardepthgreaterthan_example]
 )
+
+_SUPPORTED_DTYPES = Union[QInt, QUInt, QMontgomeryUInt]
```

Comment:
this is only used in one place (?) consider un-factoring out. Just put it inline